marlowe-apps/src/Language/Marlowe/Runtime/App/Transact.hs

```diff
@@ -170,9 +181,36 @@ transactWithEvents event config key request =
             $ \case
               TxInfo{} -> pure ()
               response -> unexpected response
+          when (confirmSeconds > 0)
+            . withSubEvent subEvent (DynamicEventSelector "WaitAfterConfirm")
+            . const
+            $ liftIO . threadDelay . (confirmSeconds +)
```

Comment:
```suggestion
            $ liftIO . threadDelay . (confirmSeconds *)
```
If I understand it right, in the help message it says a factor, so I think either this or the help is wrong. (It is adding instead of multiplying.)

Reply:
Thanks! Definitely a bug. (I had switched between multiplicative and additive approaches several times.) I'll fix and retest.

Reply:
I retested with 20-fold parallelism for a total of 200 contracts and 800 transactions: [100% success](http://testing.marlowe.run/scaling/2023012459.log.gz).